LR_Tourist/BLL/Interfaces/ICRUDService.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+using System;
+using System.Collections.Generic;
+
+namespace BLL.Interfaces
+{
+    public interface ICRUDService<T>
+        where T : class
```

Comment:
T should be a IBaseEntity

Reply:
why?

Reply:
Because your Delete method requires `Id` field.
And `T` type should have it.
IEntityBase have this requirement.